gnd/src/main/java/com/google/android/gnd/ui/home/HomeScreenFragment.java

```diff
@@ -284,7 +283,7 @@ private void onShowAddFeatureDialogRequest(Point location) {
     }
     // TODO: Pause location updates while dialog is open.
     // TODO: Show spinner?
-    showFeatureDialogRequests.onNext(getChildFragmentManager());
+    showFeatureDialogRequests.onNext(new Object());
```

Comment:
@gino-m  Can you just confirm that this is the best way to pass a dummy object downstream? Just want to make sure there's not some other particular invocation that's appropriate here regarding memory consumption etc.

Reply:
Using a new Object instance is completely fine, thought I prefer passing a reference to something meaningful. Speaking of which, I notice we don't do anything with the location passed into this method. Shouldn't that be propagated through the stream?

Reply:
P.S. The location shown when add was clicked is needed because we can't rely on the current center of the map after selecting the feature type; it might have moved due to blue-dot navigation or race condition with accidental pan,

Reply:
Historically this method wasn't doing anything with the location argument previously (as far as I can tell)--for this change I was trying to refactor for the subscriptions only without affecting other bits of functionality, but what you're saying makes sense. I'm just not certain it's actually used downstream anywhere at this point. There is a TODO to pause location updates on dialog show--maybe we should tackle that? 

Let me take a closer look at this! I think we should probably delegate it to a follow-up PR though, unless I'm wrong and my change actually modified the existing method more than I thought.

Reply:
Ok, if you're matching current behavior it makes sense as you have it now. Would just add a TODO to use the location instead.